core/src/main/java/org/polypheny/db/plan/AlgOptNode.java

```diff
@@ -74,7 +75,7 @@ public interface AlgOptNode {
     AlgTraitSet getTraitSet();
 
     // TODO: We don't want to require that nodes have very detailed row type. It may not even be known at planning time.
```

Comment:
update java doc

Reply:
Adjusted to use language-independent expressions throughout.